avocado/core/test.py

```diff
@@ -208,9 +208,9 @@ def record_and_warn(*args, **kwargs):
             params = []
         elif isinstance(params, tuple):
             params, mux_path = params[0], params[1]
-        self.params = multiplexer.AvocadoParams(params, self.name,
-                                                mux_path,
-                                                self.default_params)
+        self.params = varianter.AvocadoParams(params, self.name,
```

Comment:
My focus was solely to multiplexation, therefor I haven't tackle to AvocadoParams. We could extract them later, if needed. The reason we have to separate mux is that it's about to become semi-public interface (to allow custom varianter plugins)